src/rust-crypto/backup.ts

```diff
@@ -591,6 +603,173 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
     public createBackupDecryptor(decryptionKey: RustSdkCryptoJs.BackupDecryptionKey): BackupDecryptor {
         return new RustBackupDecryptor(decryptionKey);
     }
+
+    /**
+     * Restore a key backup.
```

Comment:
For consistency with other declarations:
```suggestion
     * Restore a key backup.
     *
```